website/docs/reference/change-requests.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Change requests
+---
+
+:::info
+
+The change requests feature was introduced in **Unleash 4.18.0**, and is currently an enterprise only feature in beta
+with a select few customers. If you're an unleash enterprise customer, please contact us if you'd like to try out this 
+feature. 
+
+:::
+
+Feature flagging is a powerful tool, and because it's powerful you sometimes need to practice caution. The ability to have complete control over your production environment comes at the cost of being able to make mistakes in production. Change requests are here to alleviate that fear. Introduced in 4.18.0, you can now group changes in a change request instead of applying them directly to production. This allows you to make changes to multiple feature toggles configuration and status (on/off) grouped together, and applied together at the same time. 
+
+Our goal is developer effeciency, but we also recognize that we have users and customers in highly regulated industries, governed by law and strict requirements. Therefore, we have added a capability to change requests that will allow you to enforce the _4 eyes principle_.
+
+## Change request configuration
+
+The change request configuration can be set up per project, per environment. This means that you can have different change request configurations for different environments. This is useful if you have different requirements for different environments, for example production and development. The implication of this is that you can't change toggles across projects in the same change request.
```

Comment:
The change request configuration can be set up per project, per environment. This means that you can have different change request configurations for different environments, such as production and development. This is useful because different environments may have different requirements, so you can customize the change request configuration to fit those requirements. However, this also means that you cannot change toggles across projects in the same change request.